CorsixTH/Lua/game_ui.lua

```diff
@@ -459,25 +459,25 @@ function GameUI:onMouseMove(x, y, dx, dy)
       (x < scroll_region_size or y < scroll_region_size or
        x >= self.app.config.width - scroll_region_size or
        y >= self.app.config.height - scroll_region_size) then
-    local dx = 0
-    local dy = 0
+    local dx2 = 0
```

Comment:
May have noticed I'm not a fan of *2 variables. Here dx, dy are change in the x / y location of the mouse between calls of ```onMouseMove``` dx2 and dy2 are the scroll distances. Silly things are done with ```tick_scroll_amount_mouse``` otherwise I'd suggest dropping the temp variable entirely. Suggesting ```tsam_x``` and ```tsam_y``` instead.